app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt

```diff
@@ -90,9 +93,11 @@ class LoginActivity : AppCompatActivity(), ILoginView {
     }
 
     override fun onLoginError(title: String?, message: String?) {
+        progressDialog.dismiss()
```

Comment:
@amitiwary999 Before calling onLoginError Method from presenter, progress dialog is dismissed here https://github.com/fossasia/susi_android/blob/development/app/src/main/java/org/fossasia/susi/ai/login/LoginPresenter.kt#L98 . I am not able to reproduce the bug. See this https://youtu.be/JivszzGFh8s